askama/Cargo.toml

```diff
@@ -39,7 +39,14 @@ mime_guess = []
 [dependencies]
 askama_derive = { version = "0.12.0", path = "../askama_derive" }
 askama_escape = { version = "0.10.3", path = "../askama_escape" }
-askama_shared = { version = "0.13.0", path = "../askama_shared", default-features = false }
+
+comrak = { version = "0.12", optional = true, default-features = false }
+humansize_ = { package = "humansize", version = "1.1.0", optional = true }
```

Comment:
Not a fan of the `_` suffixes here. Is there another way to do this?

Reply:
Yes, use a different prefix / suffix 😛

The prefix / suffix doesn't need to be visible in the code, you can rename the dependency back via `extern crate humansize_ as humansize;` ([related post on my blog](https://blog.turbo.fish/cargo-features/#reusing-the-name-of-an-optional-dependency-for-a-feature)).

The cleanest solution would be to not rename, and use `dep:humansize` syntax for the dependency of the feature on the dependency. That is however a [new feature from 1.60](https://blog.rust-lang.org/2022/04/07/Rust-1.60.0.html#new-syntax-for-cargo-features), so would raise the MSRV.

Reply:
We can rename the features instead of the dependencies, right?

Reply:
Sure, that works too. But it often causes a lot of confusion, especially when the name of the optional dependency was previously a feature name. Every optional dependency is automatically a feature (unless shadowed by an explicit feature as of 1.60), so there will be no error for activating the `humansize` feature.